website_team/doc/index.rst

```diff
@@ -10,14 +10,10 @@ Installation
 Usage
 =====
 
-**Usage**
+To make the user appear on the "Team" page, it is necessary to add his GitHub username.
```

Comment:
As I understand, user appear at team page at anyway. But with github username it will have a link to persona page